src/JWT/JWT.csproj

```diff
@@ -3,10 +3,9 @@
   <PropertyGroup>
     <TargetFrameworks>net35;netstandard1.3</TargetFrameworks>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(TargetFramework)' == 'net35'">
-    <TargetFrameworkIdentifier>.NETFramework</TargetFrameworkIdentifier>
-    <DefineConstants>NET35</DefineConstants>
-    <FrameworkPathOverride>C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v3.5\Profile\Client</FrameworkPathOverride>
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net462'">

```

Comment:
Currently it's this:

    <TargetFrameworks>net462;netstandard1.3</TargetFrameworks>

Isn't this correct?